src/json_reporter.cc

```diff
@@ -167,12 +167,24 @@ bool JSONReporter::ReportContext(const Context& context) {
   }
   out << "],\n";
 
+#if defined(BENCHMARK_VERSION)
```

Comment:
do you think this should be in a header somewhere so it can be used across the project or even accessible by integrating binaries?